docs/NetCDF-to-Zarr-Example-Usage.ipynb

```diff
@@ -0,0 +1,391 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "43fb257b-e819-44ae-b5c9-3c619a3dd065",
+   "metadata": {},
+   "source": [
+    "# NetCDF-to-Zarr service example usage\n",
+    "\n",
+    "The scope of this notebook is to demonstrate example requests that use the NetCDF-to-Zarr Harmony backend service via `harmony-py`. Once each request has completed, the output will be accessed in-region from S3.\n",
+    "\n",
+    "To run the cells that access the data in S3, this notebook will need to be run from within the AWS `us-west-2` region. One way of accomplishing this is to run the notebook within the [Openscapes 2i2c Jupyter Hub](https://openscapes.2i2c.cloud/hub).\n",
+    "\n",
+    "### Importing required packages:\n",
+    "\n",
+    "The cell below imports classes and functions from various Python packages, these packages include:\n",
+    "\n",
+    "* `harmony-py`: A package that allows for easy, interaction with the Harmony API that can be written directly in Python.\n",
+    "* `pprint`: A lightweight package that can print data structures with indentation to aid in their visual inspection.\n",
+    "* `s3fs`: A package that enables interactions with items stored in AWS S3.\n",
+    "* `matplotlib`: A package used extensively for plotting data in Python.\n",
+    "* `xarray`: A package that can read and write data stored in a number of formats, including NetCDF-4 and Zarr.\n",
+    "\n",
+    "The packages above should be installed in the environment in which this notebook is running, prior to starting the notebook. They should be installed by default in the Openscapes 2i2c Jupyer Hub as part of the Corn development environment."
```

Comment:
Corn is the name for the development environment they use in Openscapes (because it has lots of kernels, I think is the joke): https://github.com/NASA-Openscapes/corn

Reply:
Ok then.  I didn't see anything about corn when I logged in to my openscapes hub, but this is fine I suppose